core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-page-dropzone/pipes/error/dot-error.pipe.ts

```diff
@@ -37,7 +37,8 @@ export class DotErrorPipe implements PipeTransform {
             ?.container;
 
         if (
-            container?.identifier !== originContainer?.identifier && // If it is not from the same container then we are adding a new contentlet
+            (container?.identifier !== originContainer?.identifier ||
```

Comment:
Could be nice a function with a nice name to understand what means all this `or` and `and`